Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBase/ListViewExtensions.SmoothScrollIntoView.cs

```diff
@@ -56,15 +56,15 @@ public static async Task SmoothScrollIntoViewWithIndexAsync(this ListViewBase li
                 previousXOffset = scrollViewer.HorizontalOffset;
                 previousYOffset = scrollViewer.VerticalOffset;
 
-                var tcs = new TaskCompletionSource<object>();
+                var tcs = new TaskCompletionSource<VoidResult>();
 
-                void ViewChanged(object obj, ScrollViewerViewChangedEventArgs args) => tcs.TrySetResult(result: null);
+                void ViewChanged(object obj, ScrollViewerViewChangedEventArgs args) => tcs.TrySetResult(result: default);
 
                 try
                 {
                     scrollViewer.ViewChanged += ViewChanged;
                     listViewBase.ScrollIntoView(listViewBase.Items[index], ScrollIntoViewAlignment.Leading);
-                    await tcs.Task;
+                    await tcs.Task.ConfigureAwait(true);
```

Comment:
All these `ConfigureAwait(true)` calls don't actually do anything and they just create more noise, we should remove them. `ConfigureAwait(bool)` should really only ever be used with `false`, and the only reason it takes a parameter is because it might possibly be useful in some niche test scenarios where you might want to control that behavior at runtime. But `ConfigureAwait(true)` on its own is just meaningless.

Reply:
I will remove it.

My thought process: In this case, `ConfigureAwait` must be `true` otherwise my code will break. Somewhere in the guidelines or in an official blog, I saw that we should specify `ConfigureAwait` as `true` if we are purposefully not using `ConfigureAwait(false)`. Anyway, I will remove it.

Reply:
Not sure where that was from, but the official guideline is to just remove it, as it doesn't do anything.
See [this blog post](https://devblogs.microsoft.com/dotnet/configureawait-faq/#why-would-i-want-to-use-configureawaittrue) from Stephen Toub 😄

Reply:
From the blog, you point out. In my case, this must be running on the same thread (UI Thread) otherwise my code will break.

> You wouldn’t, unless you were using it purely as an indication that you were purposefully not using ConfigureAwait(false)

But I agree with you, personally, I also don't like it. Just trying to follow standards while raising a PR for Microsoft itself. 😁